.changeset/unlucky-dolphins-chew.md

```diff
@@ -0,0 +1,9 @@
+---
+"@react-router/dev": patch
+"react-router": patch
+---
+
+Disable Lazy Route Discovery for all `ssr:false` app and not just "SPA Mode" because there is no runtime server to serve the search-param-configured `/__manifest` requests
```

Comment:
```suggestion
Disable Lazy Route Discovery for all `ssr:false` apps and not just "SPA Mode" because there is no runtime server to serve the search-param-configured `__manifest` requests
```